sdk/python/lib/pulumi/runtime/resource.py

```diff
@@ -81,7 +81,6 @@ async def prepare_resource(res: 'Resource',
                            opts: Optional['ResourceOptions'],
                            typ: Optional[type] = None) -> ResourceResolverOperations:
     from .. import Output  # pylint: disable=import-outside-toplevel
-    log.debug(f"resource {props} preparing to wait for dependencies")
```

Comment:
Deleting the `log.debug` calls at the beginning/end of this function, because we don't do it in the Node.js SDK